app/configurator/configurator-state/reducer.tsx

```diff
@@ -1094,13 +1102,68 @@ export const reducer_: Reducer<ConfiguratorState, ConfiguratorStateAction> = (
 };
 
 /** Turn this on for the reducer to log state, action and result */
-const reducerLogging = false;
+const reducerLogging = true;
 const withLogging = <TState, TAction extends { type: unknown }>(
   reducer: Reducer<TState, TAction>
 ) => {
   return (state: Draft<TState>, action: TAction) => {
     const res = reducer(state, action);
+    console.log(`Action: ${action.type}`, action, res);
```

Comment:
👁️‍🗨️

Reply:
This one is OK, it's when we have activated the loggin.

Reply:
Ah, you're right 👍